locale/ca/site.json

```diff
@@ -71,11 +71,13 @@
     },
     "package-manager": {
       "link": "download/package-manager",
-      "text": "instal·lar Node.js amb gestor de paquets"
+      "text": "instal·lar Node.js mitjançant gestor de paquets"
```

Comment:
Start with capital I: `Instal·lar...`

Reply:
`mitjançant un gestor de paquets` could be more accurate